addons/scroll-project-titles/style.css

```
@@ -0,0 +1,4 @@
+.project-title.no-edit {
```

Scratch's selector is more specific, so yours will need to be the same to override the styles.
```suggestion
.preview .project-title.no-edit {
```